### PR TITLE
SONAR-7372 Too many logs "Disable metric" at server startup

### PR DESCRIPTION
**Pull request by simonbrandhof**
SONAR-7372 Too many logs "Disable metric" at server startup

### CODE REVIEW

File: server/sonar-server/src/main/java/org/sonar/server/startup/RegisterMetrics.java [hunk 1]
```diff
@@ -103,8 +103,9 @@ private void save(DbSession session, Iterable<Metric> metrics) {
 
     for (MetricDto nonUpdatedBase : basesByKey.values()) {
       if (!nonUpdatedBase.isUserManaged()) {
-        LOG.info("Disable metric {} [{}]", nonUpdatedBase.getShortName(), nonUpdatedBase.getKey());
-        dbClient.metricDao().disableCustomByKey(session, nonUpdatedBase.getKey());
+        if (dbClient.metricDao().disableCustomByKey(session, nonUpdatedBase.getKey())) {
```
Comment (SonarTech) [inline server/sonar-server/src/main/java/org/sonar/server/startup/RegisterMetrics.java]: MAJOR Merge this if statement with the enclosing one. [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1066)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Comment by sns-seb** [comment]
unless replacing disabling metrics one by one by a single statement is easy to do, LGTM

**Comment by simonbrandhof** [comment]
As discussed there is no performance issue right now so we keep the same behavior.

**Pull request closed without merging**